multicluster/cmd/gateways.go

```diff
@@ -41,7 +41,14 @@ func newGatewaysCommand() *cobra.Command {
 				return err
 			}
 
-			client, err := public.NewExternalClient(cmd.Context(), controlPlaneNamespace, k8sAPI)
+			ctx := cmd.Context()
+
+			vizNs, err := k8sAPI.GetNamespaceWithExtensionLabel(ctx, "linkerd-viz")
```

Comment:
You mean `vizChartName`? We'd have to introduce a new one in case the chart name changes. Also I see many commands in viz are still using the string instead of a const, so I'd rather leave this for general cleanup aftewards.

Reply:
I see it was me that added some of those strings after all. I'll create the constant then.